### DISCUSSION TITLE
an error of running iox-roudi

### DISCUSSION THREAD

**Discussion by qclzdh** [eclipse-iceoryx/iceoryx#877] [Q&A] [answered]
Hello, i am porting iceoryx to one Linux based OS, when i try to run iox-roudi daemon, i got following error, Please help to give me some hints, how i can fix it.

2021-07-10 12:11:40.347 [ Info  ]: No config file provided and also not found at '/etc/iceoryx/roudi_config.toml'. Falling back to built-in config.
Log level set to: [Warning]
SharedMemory still there, doing an unlink of /iceoryx_mgmt
Reserving 60138536 bytes in the shared memory [/iceoryx_mgmt]
[ Reserving shared memory successful ] 
SharedMemory still there, doing an unlink of /root
Reserving 149264720 bytes in the shared memory [/root]
[ Reserving shared memory successful ] 
/home/qianchunlei/project/imate/external/iceoryx/iceoryx/iceoryx-1.0.1/iceoryx_utils/source/posix_wrapper/access_control.cpp:66 **_{ bool iox::posix::AccessController::writePermissionsToFile(int32_t) const }  :::  [ -1 ]  Operation not supported_**
/home/qianchunlei/project/imate/external/iceoryx/iceoryx/iceoryx-1.0.1/iceoryx_utils/source/posix_wrapper/access_control.cpp:66 **_{ bool iox::posix::AccessController::writePermissionsToFile(int32_t) const }  :::  [ 95 ]  Operation not supported_**
_**Error: Could not set file ACL.**_
2021-07-10 12:11:44.598 [ Error ]: ICEORYX error! MEPOO__SEGMENT_COULD_NOT_APPLY_POSIX_RIGHTS_TO_SHARED_MEMORY
terminate called without an active exception
Aborted

**Comment by elBoberido** [comment, marked as answer]
@qclzdh the problem is that you have `libacl` on your system but they are either just stubs or RouDi is not privileged for the calls. From the errror messages it is more probable that the lib contains just stubs. Do you have access to a working `libacl`?

**Comment by qclzdh** [comment]
Thanks, i fixed this by add following config in my kernel, maybe this can be added in some doc~
**_CONFIG_TMPFS_POSIX_ACL=y
CONFIG_TMPFS_XATTR=y_**
